changelog/261.improvement.md

```diff
@@ -0,0 +1,5 @@
+Added an optional `ignored_intents` parameter in forms.
+
+- In order to use it, please add in your `domain.yml` file the `ignored_intents` parameter with a value/list of values of your choice after the form's name. Please see [Forms](forms.mdx) for more information.
+- All slot mappings of a form won't apply if the intent of a message is specific in `ignored_intents`.
+- This can be used in case the user never wants to fill any slots of a form with the specified intent, eg. chitchat. 
```

Comment:
```suggestion
- To use it, add the `ignored_intents` parameter  in your `domain.yml` file after the forms name and provide a list of intents to ignore. Please see [Forms](forms.mdx) for more information.
- This can be used in case the user never wants to fill any slots of a form with the specified intent, eg. chitchat. 
```

Reply:
Since it's more completely described in the docs, shortened the changelog a bit